modules/enumeratum/src/main/scala/pureconfig/module/enumeratum/package.scala

```diff
@@ -11,47 +11,47 @@ import pureconfig.error.CannotConvert
 
 package object enumeratum {
 
-  implicit def enumeratumConfigConvert[A <: EnumEntry](implicit enum: Enum[A], ct: ClassTag[A]): ConfigConvert[A] =
-    viaNonEmptyStringOpt[A](enum.withNameOption, _.entryName)
+  implicit def enumeratumConfigConvert[A <: EnumEntry](implicit `enum`: Enum[A], ct: ClassTag[A]): ConfigConvert[A] =
```

Comment:
Rather than quoting all the local variables named "enum" here, why don't you rename it to something else, like `e` or `en`? It leaves the code cleaner and closer to the way one would write it for Scala 3.

Reply:
will do so